Query and Updates

Wikitelmex opened the pull request:
- add new animal records
- add new Column to table animals
- add several queries (description of each one in code comments)

wikitelmex1 committed:
Query and Updates

- add new animal records
- add new Column to table animals
- add several queries (description of each one in code comments)

oxenprogrammer commented on queries.sql:
```diff
@@ -8,3 +8,93 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
+*/
+BEGIN TRANSACTION;
+	UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+	UPDATE animals SET species='pokemon' WHERE name NOT LIKE '%mon';
```
**Change Required**
- [x] UPDATE the animals table by setting the species column to pokemon for all animals that don't have species already set. Currently this will set `pokemon` as long as it doesn't end with `mon`. But the requirement dictates that `all animals that don't have species already set`. Let us try and fix this.

Wikitelmex replied:
All right, I fixed that.

oxenprogrammer reviewed (changes requested):
Hi @Wikitelmex , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wikitelmex1 committed:
Changes Required 1

- Change UPDATE animals to set all the species that doesn't have any value to 'pokemon'

descholar-ceo reviewed (approved):
Hi @Wikitelmex , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
